pkg/config/config.go

```diff
@@ -53,9 +53,9 @@ type Config struct {
 	// expression for its argument.
 	ShowLocationExpr bool `yaml:"show-location-expr"`
 
-	// Source list line-number color (3/4 bit color codes as defined
-	// here: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors)
-	SourceListLineColor int `yaml:"source-list-line-color"`
+	// Style to be used to highlight source code as defined
+	// here: https://godoc.org/github.com/alecthomas/chroma#Style
+	SourceCodeStyle string `yaml:"source-code-template"`
```

Comment:
Having the ability to specify the base color for the source code was a feature added by someone. I don't use it but I think they probably liked it and I would rather not remove it. I'm not sure how the themes work here, but it seems moot given that there is only one.

Reply:
So far the So far [packaga]() has only one theme but I think in time more can be added. If you don't like this restriction, do you have any suggestions to replace it?

Reply:
There hasn't been any updated here in a while. @aarzilli do you have a response to this?

Reply:
I still think we shouldn't remove the option for specifying the default color.